opencti-platform/opencti-front/src/private/components/analysis/malware_analyses/MalwareAnalysesLines.tsx

```diff
@@ -0,0 +1,106 @@
+import { MalwareAnalysesLines_data$key } from './__generated__/MalwareAnalysesLines_data.graphql';
+
+const nbOfRowsToLoad = 50;
+
+interface MalwareAnalysesLinesProps {
+  paginationOptions?: MalwareAnalysesLinesPaginationQuery$variables,
+  dataColumns: DataColumns,
+  queryRef: PreloadedQuery<MalwareAnalysesLinesPaginationQuery>,
+  setNumberOfElements: UseLocalStorageHelpers['handleSetNumberOfElements'],
+}
+
+export const malwareAnalysesLinesQuery = graphql`
+    query MalwareAnalysesLinesPaginationQuery(
+        $search: String
+        $count: Int
+        $cursor: ID
+        $orderBy: MalwareAnalysesOrdering
+        $orderMode: OrderingMode
+        $filters: [MalwareAnalysesFiltering!]
+    ) {
+        ...MalwareAnalysesLines_data
+        @arguments(
+            search: $search
+            count: $count
+            cursor: $cursor
+            orderBy: $orderBy
+            orderMode: $orderMode
+            filters: $filters
+        )
+    }
+`;
+
+const malwareAnalysesLinesFragment = graphql`
+    fragment MalwareAnalysesLines_data on Query
+    @argumentDefinitions(
+        search: { type: "String" }
+        count: { type: "Int" }
+        cursor: { type: "ID" }
+        orderBy: { type: "MalwareAnalysesOrdering" }
+        orderMode: { type: "OrderingMode" }
+        filters: { type: "[MalwareAnalysesFiltering!]" }
+    ) @refetchable(queryName: "MalwareAnalysesLinesRefetchQuery") {
+        malwareAnalyses(
+            search: $search
+            first: $count
+            after: $cursor
+            orderBy: $orderBy
+            orderMode: $orderMode
+            filters: $filters
+        ) @connection(key: "Pagination_malwareAnalyses") {
+            edges {
+                node {
+                    id
+                    product
+                    submitted
+                    ...MalwareAnalysisLine_node
+                }
+            }
+            pageInfo {
+                endCursor
+                hasNextPage
+                globalCount
+            }
+        }
+    }
+`;
+
+const MalwareAnalysesLines: FunctionComponent<MalwareAnalysesLinesProps> = ({ setNumberOfElements, dataColumns, queryRef, paginationOptions }) => {
+  const {
+    data,
+    hasMore,
+    loadMore,
+    isLoadingMore,
+  } = usePreloadedPaginationFragment<MalwareAnalysesLinesPaginationQuery, MalwareAnalysesLines_data$key>({
+    linesQuery: malwareAnalysesLinesQuery,
+    linesFragment: malwareAnalysesLinesFragment,
+    queryRef,
+    nodePath: ['malwareAnalyses', 'pageInfo', 'globalCount'],
+    setNumberOfElements,
+  });
+  return (
+    <ListLinesContent
+      initialLoading={!data}
+      hasMore={hasMore}
+      loadMore={loadMore}
+      isLoading={isLoadingMore}
+      dataList={data?.malwareAnalyses?.edges ?? []}
+      globalCount={data?.malwareAnalyses?.pageInfo?.globalCount ?? nbOfRowsToLoad}
```

Comment:
This seems strange, I'll check on that as it is done everywhere.

Reply:
It is done the same way almost everywhere

Reply:
After checking, this is useless, remove it please